test/test_kfac.py

```diff
@@ -436,17 +440,26 @@ def test_multi_dim_output(
     # set up loss function, data, and model
     loss_func = loss(reduction=reduction).to(dev)
     loss_average = None if reduction == "sum" else "batch+sequence"
+    X1 = rand(2, 7, 5, 5)
+    X2 = rand(4, 7, 5, 5)
     if isinstance(loss_func, MSELoss):
         data = [
-            (rand(2, 7, 5, 5), regression_targets((2, 7, 5, 3))),
-            (rand(4, 7, 5, 5), regression_targets((4, 7, 5, 3))),
+            (X1, regression_targets((2, 7, 5, 3))),
+            (X2, regression_targets((4, 7, 5, 3))),
+        ]
+        manual_seed(711)
+        model = Sequential(Linear(5, 4), Linear(4, 3)).to(dev)
+    elif issubclass(loss, BCEWithLogitsLoss):
```

Comment:
Is this `issubclass` intentional, or should this be `isinstance`?

Reply:
It is intentional, since the class is passed as an argument and not a class instance.